statsbombpy/helpers.py

```diff
@@ -25,7 +59,7 @@ def flatten_event(event, flatten_attrs):
 def filter_and_group_events(events, filters, fmt, flatten_attrs):
     events_ = defaultdict(list)
     for ev in events.values():
-        ev_type = pluralize(ev["type"]["name"])
+        ev_type = PLURALS.get(ev["type"]["name"])
```

Comment:
I think you want it to fail rather than silently rename it to None if a new ev_type appears.

Reply:
Good point

Reply:
What if we set it so that it doesn't pluralise the event if it can't find it in the dictionary? Like:

`ev_type = PLURALS.get(ev["type"]["name"], ev["type"]["name"])`

or if we still want the error would this be ok?:

```
ev_type = PLURALS.get(ev["type"]["name"])
if ev_type is None:
    raise ValueError(f'{ev["type"]["name"]}: event not found in mapping dict')
```

Reply:
If it failed, wouldn't that basically break the whole library?

Reply:
Yeah the event functions wouldn't work.

Reply:
This feels like making it more complex than necessary. I think you either pluralize every ev_type or you get them from the dictionary, and if the ev_type is not in the dictionary, it already fails with KeyError